### PR TITLE
Removes Hippiestation 13

### PR DESCRIPTION
**Pull request by yobob591**
remove

### CODE REVIEW

**Comment by Spacedong** [comment]
I mean honestly if you ask me its probably for the best

**Comment by Hackole** [comment]
end it all +1

**Comment by ArcLumin** [comment]
This is a bugfix, it will fix all the bugs. Code's good.

**Comment by Spacedong** [comment]
rip conflicts

**Comment by yobob591** [comment]
fuuck i forgot to remove tgstation

**Comment by Spacedong** [comment]
should probably do that for travis to pass, then one more headmin can +1 and we can finally get off this shitshow

**Comment by QuarianCommando** [comment]
So what happens if someone merges it?

**Comment by yobob591** [comment]
it removes hippiestation 13 and we are done

**Comment by Hackole** [comment]
merge it; and deliver us from this endless hell we find ourselves in

**Comment by yobob591** [comment]
im going ot fucking dismember this server

**Comment by SpankMasterson** [comment]
+1

**Comment by Spacedong** [comment]
Ready to merge.

**Comment by TDcoolguy300** [comment]
?Fix it until Travis doesn't explain

**Comment by TDcoolguy300** [comment]
No merge for you then

File: SQL/database_changelog.txt [hunk 1]
```diff
@@ -1,91 +0,0 @@
...
-Modified table 'watch', removing 'id' column, making 'ckey' primary and adding the columns 'timestamp', 'adminckey', 'last_editor' and 'edits'.
-
-ALTER TABLE `feedback`.`watch` DROP COLUMN `id`, ADD COLUMN `timestamp` datetime NOT NULL AFTER `reason`, ADD COLUMN `adminckey` varchar(32) NOT NULL AFTER `timestamp`, ADD COLUMN `last_editor` varchar(32) NULL AFTER `adminckey`, ADD COLUMN `edits` text NULL AFTER `last_editor`, DROP PRIMARY KEY, ADD PRIMARY KEY (`ckey`)
-
-Remember to add a prefix to the table name if you use them.
-
-----------------------------------------------------
-
-14 August 2015, by Jordie0608
-
```
Comment (TDcoolguy300) [inline SQL/database_changelog.txt:12]: Fix Travis first

**Comment by TDcoolguy300** [comment]
Fix Travis or no merge

**Comment by TDcoolguy300** [comment]
Closing it until you fix Travis

**Comment by yobob591** [comment]
tell me whats wronggg with traorbis

**Comment by ArcLumin** [comment]
chmod: cannot access `tools/check_filedirs.sh': No such file or directory

**Comment by ArcLumin** [comment]
you deleted travis' tools now he's pissed

**Comment by yobob591** [comment]
oh no

**Comment by ArcLumin** [comment]
face travis' wrath or suffer the consequences

**Comment by TDcoolguy300** [comment]
No fix? This PR has been banished from ever returning, all future PRs of this type will be quickly closed

**Pull request closed without merging**